documentation/docs/developers/connectors/task4.md

```diff
@@ -1,9 +1,11 @@
 # Task 4. Required Connector Configuration
 
-This section explains the required connector configuration in order to work with Hummingbot.
+This section explains the required steps for connector in order to work with Hummingbot.
 
-First, add your new connector package to `setup.py`, in `packages` variable add your package as below:
+To do so:
 
+1. Add your new connector package to `setup.py`.
+2. In `packages` variable, add in your package as shown:
```

Comment:
This is actually just one step, the `packages` variable is in `setup.py` file. The first step is not needed.